aws/resource_aws_rds_cluster.go

```diff
@@ -398,6 +398,7 @@ func resourceAwsRDSCluster() *schema.Resource {
 						"error",
 						"general",
 						"slowquery",
+						"postgresql",
```

Comment:
Should add `upgrade` also, IMHO.

Reference:

> '{"EnableLogTypes":["postgresql", "upgrade"]}'
https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.CloudWatch.html 'AWS CLI' section

https://github.com/terraform-providers/terraform-provider-aws/pull/6829/files#diff-1c09d2c3018128a511db5f6dcde728d8R411

Reply:
Hi saiya,

I'm not sure if the documentation is wrong at this point, but when I try to add upgrade via the cli I receive the following error

> An error occurred (InvalidParameterCombination) when calling the CreateDBCluster operation: You cannot use the log types 'upgrade' with engine version aurora-postgresql 10.7. For supported log types, see the documentation.

Reply:
Understood. And I confirmed only `postgresql` log type is exportable in AWS Console (web UI).

https://github.com/terraform-providers/terraform-provider-aws/pull/6829#issuecomment-519236795
> "ExportableLogTypes": [
>                "postgresql"
>            ],

Only `postgresql` seems to be enough at this moment.